docs/core/preview3/tools/csproj.md

```diff
@@ -13,12 +13,10 @@ ms.assetid: bdc29497-64f2-4d11-a21b-4097e0bdf5c9
 
 # Additions to csproj format for .NET Core
 
-This document outlines the changes that were added to the csproj files as part of the move from project.json to csproj and 
-[MSBuild](https://github.com/Microsoft/MSBuild). This document outlines **only the deltas to non-core csproj files**. If 
-you need more information about general project file syntax and reference, please consult [the MSBuild project file]() documentation. 
```

Comment:
@blackdwarf did you have a particular topic in mind? for now, I just removed the link.